analyzer/src/main/kotlin/curation/ClearlyDefinedPackageCurationProvider.kt

```diff
@@ -91,7 +91,7 @@ class ClearlyDefinedPackageCurationProvider(server: Server = Server.PRODUCTION)
         } catch (e: HttpException) {
             e.showStackTrace()
 
-            log.warn { "Getting curations for '${pkgId.toCoordinates()}' failed with: ${e.collectMessagesAsString()}" }
+            log.debug { "Getting curations for '${pkgId.toCoordinates()}' failed with: ${e.collectMessagesAsString()}" }
```

Comment:
Not sure whether it is overkill, but would it make sense to evaluate the response status and treat a 404 differently? It seems to be a regular return code and not actually an error condition, so one could argue that there is no need to log anything. OTOH, a different error might still deserve a log statement on a higher level as it is unexpected.

Reply:
If the ticket linked in the commit message was implemented, I believe a debug log is fine.
If they don't implement it then IMO it would make sense to remove the log statement.
IMO we could keep it as is to avoid some back and forth and just assume the ticket will be implemented (soon).

Reply:
> would it make sense to evaluate the response status and treat a 404 differently?

Good idea, done.